README.md

```diff
@@ -76,3 +76,11 @@ In summary execute the following:
 If there is a problem, you can rollback using:
 
     mvn release:rollback
+
+EPTS Training example
```

Comment:
This is what it looks like if I comment on some of your changes.

Reply:
got it,  its amazing.